assets/javascripts/discourse/connectors/user-preferences-security/encrypt.hbs

```diff
@@ -17,39 +17,42 @@
             {{#if isEncryptActive}}
               <p>{{i18n "encrypt.preferences.status_enabled"}}</p>
               <fieldset class="control-group">
-                {{d-button
-                  icon="times"
-                  action=(action "deactivateEncrypt")
-                  label="encrypt.preferences.deactivate"
-                  id="encrypt-deactivate"
-                }}
-                {{d-button
-                  id="encrypt-generate-paper-key-for-device"
-                  icon="plus"
-                  action=(action "generatePaperKey" true)
-                  label="encrypt.generate_paper_key.title_device"
-                }}
-                {{d-button
-                  id="encrypt-generate-paper-key"
-                  icon="ticket-alt"
-                  action=(action "generatePaperKey")
-                  label="encrypt.generate_paper_key.title"
-                }}
-                {{encrypt-preferences-dropdown
-                  id="encrypt-preferences"
-                  onChange=(action "selectEncryptPreferencesDropdownAction")
-                }}
+                <DButton
+                  @icon="times"
+                  @action={{action "deactivateEncrypt"}}
+                  @label="encrypt.preferences.deactivate"
+                  @id="encrypt-deactivate"
```

Comment:
`id` attr for d-button can be plain a html attr (and it's preferred)
See: https://github.com/discourse/discourse/blob/main/app/assets/javascripts/discourse/app/components/d-button.hbs#L9-L9

```suggestion
                  id="encrypt-deactivate"
```